lib/src/matchers/jest_dom/is_focused.dart

```diff
@@ -79,7 +77,7 @@ class _IsFocused extends Matcher {
       return mismatchDescription..add(notAnElementMismatchDescription);
     }
 
-    if (!document.documentElement.contains(item as Element)) {
+    if (!document.documentElement!.contains(item)) {
```

Comment:
I'm a little confused why so many of these Dart core types are nullable out of the box... are there any writeups / any guidance on why that is?

Reply:
That's a good question - @greglittlefield-wf do you know why that is?

Reply:
AFAIK Dart HTML api is generated form the Blink/Web IDL ... so for example documentElement is declared nullable here https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/document.idl;l=48?q=document%20idl&ss=chromium

Reply:
`document.body` is nullable because it'll be null for scripts executed before the `body` element has loaded (e.g., a script tag running in `<head>`).

For `.documentElement`: https://stackoverflow.com/a/64029469